### DISCUSSION TITLE
Configure both trackpad three finger click & three finger tap

### DISCUSSION THREAD

**Discussion by rxhanson** [rxhanson/Middle-Localization#48] [Show and tell]
The UI is purposefully really simple. To configure both the three finger click & three finger tap gestures for middle click, execute the following Terminal commands and restart the app:

`defaults write com.knollsoft.Middle trackpadThreeClick -bool true`
`defaults write com.knollsoft.Middle trackpadThreeTap -bool true`

**Comment by maximetinu** [comment]
> The UI is purposefully really simple. To configure both the three finger click & three finger tap gestures for middle click, execute the following Terminal commands and restart the app:
> 
> 
> 
> `defaults write com.knollsoft.Middle trackpadThreeClick -bool true`
> 
> `defaults write com.knollsoft.Middle trackpadThreeTap -bool true`
> 
> 

That's awesome, I'm setting that up right now 😋 Thank you very much for the quick response! 🙂

**Comment by matthewem** [comment]
This was posted 3 years ago, confirming you can get both three finger click and tap by running these Terminal commands. With that being said, I do with there was a simple option within the settings to set this up to.

**Reply by rxhanson** [reply]
Thanks for the feedback. I'm open to making changes like this, but it's hard to gauge how many people desire it, hence why I err on the side of simplicity. If more people want it, let me know!
